.zshrc

```diff
@@ -1,7 +1,6 @@
 # aliases
 alias grep="/usr/local/opt/grep/bin/ggrep"
 alias logs="jq . service.log | less"
-alias clean="docker rm -fv $(docker ps -aq)"
```

Comment:
Currently no `Docker for Mac` setup in repo, so this alias will fail looking for `docker`